nixos/lib/make-options-doc/default.nix

```diff
@@ -1,19 +1,87 @@
-/* Generate JSON, XML and DocBook documentation for given NixOS options.
+/**
+  Generates documentation for nix modules.
 
-   Minimal example:
+  # Outputs
 
-    { pkgs,  }:
+  This function offers the following outputs
 
-    let
-      eval = import (pkgs.path + "/nixos/lib/eval-config.nix") {
-        baseModules = [
-          ../module.nix
-        ];
-        modules = [];
-      };
-    in pkgs.nixosOptionsDoc {
-      options = eval.options;
+  ## optionsCommonMark
+
+  Documentation in markdown text format.
+
+  ## optionsJSON
+
+  JSON format suitable for further automated processing.
+
+  Used as an input to derive the other outputs.
+
+  ## optionsDocBook
+
+  deprecated since 23.11 and will be removed in 24.05"
+
+  Documentation rendered as DocBook from markdown format.
+
+  Rendering is done using `nixos-render-docs`.
+
+  ## optionsAsciiDoc
+
+  Documentation rendered as AsciiDoc. This is usefull for e.g. manpages.
```

Comment:
"useful", one "l". Also probably "man pages"? Not sure what the right spelling here would be.